windows_docker_resources/README.md

```diff
@@ -0,0 +1,69 @@
+## Contributing changes to Windows ROS 2 Chef Cookbook
+
+Running the Windows CI agent as a configurable chef cookbook follows the general benefits of "infrastructure as software". 
+Making changes can be submitted as pull requests, which in turn can be reviewed and tested before they are deployed.
+Below is the recommended process for submitting changes to the ROS 2 chef cookbook for Windows for use with ci.ros2.org.
+
+### Step 1. Create a ros2-cookbooks PR 
+Open a PR at github.com/ros-infrastructure/ros2-cookbooks for your desired change.
+
+### Step 2. Create a ros2/ci PR 
+Open corresponding PR at github.com/ros2/ci which updates the ros2-cookbooks git submodule to your new branch of the ros2-cookbooks repo.
+
+### Step 3. Verify the ros2-cookbooks PR 
+It can take a while to test changes to the cookbook directly on CI, so it is strongly advised to test locally and verify your change inside a representative docker container.
+See the following section for information on local testing.
+After you have tested it locally, submit jobs to https://ci.ros2.org/job/ci_windows and set CI_SCRIPTS_BRANCH to your ros2/ci PR branch.
+You may need to verify the PR for several different ROS 2 releases.
+You can set `CI_ROS_DISTRO` on ci.ros2.org, which will choose the corresponding chef cookbook configuration to test your updates.
+If testing on different releases, don't forget to specify the correct ros2.repos file.
+
+#### Testing locally
+Do the following on your own machine or VM.
+
+Get the Windows release version with the following powershell command
+```
+powershell $(Get-ItemProperty -Path 'HKLM:SOFTWARE\Microsoft\Windows NT\CurrentVersion\Update\TargetingInfo\Installed\Server.OS.amd64' -Name Version).Version
+```
+
+Change your directory to ci repo directory and run the following docker build command.
+Replace the value of WINDOWS_RELEASE_VERSION with the string you found above.
+```
+docker build  --build-arg WINDOWS_RELEASE_VERSION=10.0.18363.900 -t ros2_windows_ci -f windows_docker_resources\Dockerfile windows_docker_resources
```

Comment:
Nit: This part is not needed anymore, since there is only one `Dockerfile` again. Keep it if you like, but I thought I'd point it out.
`-f windows_docker_resources\Dockerfile`

Reply:
Removed the '-f' option.